src/main/java/com/dynatrace/oneagent/sdk/impl/noop/IncomingWebRequestTracerNoop.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2018 Dynatrace LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.dynatrace.oneagent.sdk.impl.noop;
+
+import com.dynatrace.oneagent.sdk.api.IncomingWebRequestTracer;
+
+public class IncomingWebRequestTracerNoop extends NodeNoop implements IncomingWebRequestTracer {
+
+	public static final IncomingWebRequestTracer INSTANCE = new IncomingWebRequestTracerNoop();
+	
+	private IncomingWebRequestTracerNoop() {
+	}
+	
+    @Override
```

Comment:
nit: spaces/tabs mixup

Reply:
will do codeformat in its own branch - a little bit later